app/assets/javascripts/discourse/app/templates/static.hbs

```diff
@@ -6,30 +6,32 @@
         <PluginOutlet @name="above-static" @connectorTagName="div" />
       </span>
 
-      {{html-safe this.model.html}}
+      <div class="login-welcome">
```

Comment:
@ellaestigoy this template is used for faq/guidelines/tos/privacy/etc pages. did you want to wrap their contents in `.login-welcome`?

Reply:
Nah - only for this, as the CSS styling (rounded corners, box-shadow etc) is applied to this class